### ISSUE TITLE
Determine the specific meaning and use of the number next to `ESPPD` in W2 Box 14 Other

### ISSUE THREAD

**Issue by hickeng** [hickeng/financial#91]
There's a numeric value associated with the ESPPD notation in Box 14 other, but for the example I have I'm unable to infer the meaning of the number.

This issue is to determine the meaning of the value and how to correlate it against the ordinary income portion calculated in the sheet.

**Comment by inin2626** [comment]
@hickeng Just curious, do you have ESPP listed on Box 14 from 2023 W2? I no longer work at vmw/brcm, but all of my previous colleagues said they don't have ESPP listed on W2 (they didn't sell any, they let the cash+stock conversion happen).  How can they find out whether or not VMW reported the ESPP discount on box 1 (as part of the gross income)?

By the way, I enjoy reading all the stuffs on your page. This is amazing!

**Comment by hickeng** [comment]
I didn't sell ESPP so not a good reference. I have not had anyone say that they _did_ receive an ESPP entry on W2 for either sale of fractional ESPP during the merger or sale of ESPP after, so I'm working on the assumption that the merger basically broke that integration.

> How can they find out whether or not VMW reported the ESPP discount on box 1 (as part of the gross income)?
It might show up in the granular payslips. TBH the only way I know of is to look for that ESPPD code in Box 14 but I hope it presents somewhere else as well.

> By the way, I enjoy reading all the stuffs on your page. This is amazing!

Thanks! It's been a "interesting" learning experience.... and I'm very glad to get community engagement and not have to grind through it alone. The peer review has been massively valuable for refinement and catching errors.

**Comment by inin2626** [comment]
I read some documents (from my current employee) about "when will I be subject to tax on ESPP shares". It says the discount benefit is recognized as income on W2 only when you dispose of the shares. My guess as to why there wasn't any ESPP entry on W2 is because the conversion wasn't considered as disposition... so no income of ESPP was recognized on 2023 W2

**Comment by hickeng** [comment]
> I read some documents (from my current employee) about "when will I be subject to tax on ESPP shares". It says the discount benefit is recognized as income on W2 only when you dispose of the shares. My guess as to why there wasn't any ESPP entry on W2 is because the conversion wasn't considered as disposition... so no income of ESPP was recognized on 2023 W2

Correct. The merger did not count as a disposition - the specific sections of the tax code that detail this are noted [here](https://github.com/hickeng/financial?tab=readme-ov-file#espp).

In the context of the merger, you _should_ see ESPP imputed income if:
* your fractional share was sold from an ESPP lot
* you sell ESPP shares post-merger

Normally I'd trust the automation but in this case, when selling ESPP, adding a note to yourself for tax filing season to make sure you got a W2 from Broadcom and if not to report it manually.
